src/components/OptionRow.js

```diff
@@ -101,233 +101,218 @@ const defaultProps = {
+                                                accessibilityRole={CONST.ACCESSIBILITY_ROLE.CHECKBOX}
+                                                accessibilityLabel={CONST.ACCESSIBILITY_ROLE.CHECKBOX}
+                                            >
+                                                <SelectCircle isChecked={props.isSelected} />
+                                            </PressableWithFeedback>
+                                        )}
+                                    </>
+                                )}
+                                {props.isSelected && props.highlightSelected && (
+                                    <View style={styles.defaultCheckmarkWrapper}>
+                                        <Icon
+                                            src={Expensicons.Checkmark}
+                                            fill={themeColors.iconSuccessFill}
+                                        />
+                                    </View>
+                                )}
+                            </View>
+                        </View>
+                        {Boolean(props.option.customIcon) && (
+                            <View
+                                style={[styles.flexRow, styles.alignItemsCenter]}
+                                accessible={false}
+                            >
+                                <View>
+                                    <Icon
+                                        src={lodashGet(props.option, 'customIcon.src', '')}
+                                        fill={lodashGet(props.option, 'customIcon.color')}
+                                    />
                                 </View>
-                            )}
-                        </PressableWithFeedback>
-                    )}
-                </Hoverable>
-            </OfflineWithFeedback>
-        );
-    }
+                            </View>
+                        )}
+                    </PressableWithFeedback>
+                )}
+            </Hoverable>
+        </OfflineWithFeedback>
+    );
 }
 
 OptionRow.propTypes = propTypes;
 OptionRow.defaultProps = defaultProps;
 
-export default withLocalize(OptionRow);
+export default React.memo(
+    withLocalize(OptionRow),
+    (prevProps, nextProps) =>
+        prevProps.isDisabled === nextProps.isDisabled &&
+        prevProps.isMultilineSupported === nextProps.isMultilineSupported &&
+        prevProps.isSelected === nextProps.isSelected &&
+        prevProps.shouldHaveOptionSeparator === nextProps.shouldHaveOptionSeparator &&
+        prevProps.selectedStateButtonText === nextProps.selectedStateButtonText &&
+        prevProps.showSelectedState === nextProps.showSelectedState &&
+        prevProps.highlightSelected === nextProps.highlightSelected &&
+        prevProps.showTitleTooltip === nextProps.showTitleTooltip &&
+        !_.isEqual(prevProps.option.icons, nextProps.option.icons) &&
```

Comment:
This was overlooked. `!` should have been removed. Later it caused regression - https://github.com/Expensify/App/issues/33353